.circleci/config.yml

```diff
@@ -1,13 +1,16 @@
-version: 2
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.1.0
 jobs:
   build:
     docker:
-      - image: circleci/node:8.11-browsers
+      - image: cimg/node:14.17.0-browsers
```

Comment:
What is `cimg` and why is it better than the `circleci/node` images? 

Most Angular repos have standardized on NodeJS `14.16.1` as supported by @josephperrott. Should we use that version here as well?

Reply:
`cim/node` is the newer image "designed to supercede the legacy CircleCI Node.js image, `circleci/node`"

Though I agree that ideally we should use `14.16.1` here unless we have a compelling reason to use `14.17.0`